tests/python/common/ops/test_ops.py

```diff
@@ -290,40 +288,6 @@ def test_sddmm(g, shp, lhs_target, rhs_target, msg, idtype):
         g.edata.pop("m")
 
 
-@pytest.mark.parametrize("g", get_cases(["clique"]))
-@pytest.mark.parametrize("norm_by", ["src", "dst"])
-@pytest.mark.parametrize("shp", edge_softmax_shapes)
-@parametrize_idtype
-def test_edge_softmax(g, norm_by, shp, idtype):
```

Comment:
Moved to tests/python/common/ops/test_edge_softmax.py, together with edge_softmax_hetero.